fuzz/fuzz_targets/router.rs

```diff
@@ -38,21 +38,22 @@ fuzz_target!(|data: &[u8]| {
     debug!("{}", generated_operation);
```

Comment:
nit: This is hard to leave a "suggestion" on, but the line above this should probably say "operation" or "executable document" instead of "document".

Reply:
fixed